lib/faye-redis-sharded.js

```diff
@@ -407,14 +407,14 @@ Engine.prototype = {
 
     // Try to get more information on when/why we're missing clientId by dumping the context
     try {
-      const msg = `clientId is undefined, context is: ${context}`;
+      const msg = `clientId is undefined, context is: ${JSON.stringify(context)}`;
       console.warn(msg);
       sentry.logError(new Error(msg));
     }
     finally {
       console.info("Returning default shard's Redis");
       const shards = this._shardManagers[0]._shards;
-      if (shards?.length || 0 === 0)
+      if ((shards?.length || 0) === 0)
```

Comment:
This was always evaluating to `true` so we got the error "Unable to find default shard" when it's 99.99% there 🤦